kuma/users/views.py

```diff
@@ -467,14 +470,53 @@ def user_edit(request, username):
 
             return redirect(edit_user)
 
+    subscription_info = None
+    stripe_customer = get_stripe_customer(edit_user)
+    if stripe_customer:
+        stripe_subscription_info = get_stripe_subscription_info(stripe_customer)
+        if stripe_subscription_info:
+            source = stripe_customer.default_source
+            if source.object == "card":
+                card = source
+            elif source.object == "source":
+                card = source.card
+            else:
+                raise ValueError(
+                    f"unexpected stripe customer default_source of type {source.object!r}"
+                )
+
+            subscription_info = {
+                "next_payment_at": datetime.fromtimestamp(
+                    stripe_subscription_info.current_period_end
+                ),
+                "brand": card.brand,
+                "expires_at": f"{card.exp_month}/{card.exp_year}",
+                "last4": card.last4,
+                # Cards that are part of a "source" don't have a zip
+                "zip": card.get("address_zip", None),
+            }
+
+            # To perfect the synchronization, take this opportunity to make sure
+            # we have an up-to-date record of this.
+            UserSubscription.set_active(edit_user, stripe_subscription_info.id)
```

Comment:
Curious if we might want to record [stripe's subscription created](https://stripe.com/docs/api/subscriptions/object#subscription_object-created) date in our DB as well?

Reply:
I don't see that. The `UserSubscription` record (unless it's `canceled`) means the user is an active subscriber. I.e. that they're actively paying us $$$. 
The `UserSubscription.created` is already a record of when the subscription started.